pkg/registry/networking/networkpolicy/strategy.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
I don't think we're suppose to edit copy right dates except for new files

Reply:
I'm trying to figure out why the hell I've changed this....Usually I don't...

I would blame vscode, but I don't think it would do that, so I'll take the blame and change back